docs/source/user-guide/concepts/environments.rst

```diff
@@ -52,4 +52,90 @@ environment:
 |
 
 Other conda environments usually contain the same subdirectories
-as the default environment.
\ No newline at end of file
+as the default environment.
+
+Virtual environments
+====================
+
+A virtual environment is a tool that helps to
+keep dependencies required by different projects
+separate by creating isolated Python virtual environments
+for them.
+
+Users can create virtual environments 
+using one of several tools such as
+Pipenv or Poetry, or a conda virtual
+environment. Pipenv and Poetry are based around Python's
+built-in venv library, whereas conda has its own notion of virtual
+environments that is lower-level (Python itself is a dependency provided
+in conda environments).
+
+Some other traits are:
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+
+   * - 
+     - Python virtual environment
+     - Conda virtual environment
+   * - **Libraries**
+     - Statically link, vendor libraries in wheels,
+       or use apt/yum/brew/etc.
+     - Install system-level libraries as conda dependencies
+   * - **System**
+     - Depend on base system install of Python
+     - Python is independent from system
+   * - **Extending environment**
+     - Extend environment with pip
+     - Extended environment with conda or pip
+   * - **Non-Python dependencies**
+     -
```

Comment:
extra dash here?

Reply:
oh wait, sorry, it's because there is no pip equivalent.

Reply:
So no changes?

Reply:
Yeah, no changes here.